agents/bundler-ruby/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM jetbrains/teamcity-minimal-agent:latest
+
```

Comment:
Also, you may add your `MAINTAINER` email

Reply:
Agree. I've added `LABEL` instead of `MAINTAINER` as the latter is deprecated.